libs/wingsdk/src/core/app.ts

```diff
@@ -283,8 +290,32 @@ export abstract class App extends Construct implements IApp {
     if (!type) {
       return undefined;
     }
+
     return new type(scope, id, ...args);
   }
+
+  /**
+   * Synthesize the root construct if one was given. If this is a test environment, then
+   * we will synthesize one root construct per test. Otherwise, we will synthesize exactly
+   * one root construct.
+   *
+   * @param props The App props
+   * @param testRunner The test runner
+   */
+  protected synthRoots(props: AppProps, testRunner: TestRunner) {
```

Comment:
Where is this used?

Reply:
resolved a conflict in the wrong way, thanks!